hy/core/shadow.hy

```diff
@@ -138,11 +138,17 @@
 (defn not-in [x y]
   (not-in x y))
 
+(defn get [thing key1 &rest keys]
```

Comment:
The name `thing` is kind of vague. I think we're using the term `coll` as short for "collection" like Clojure does elsewhere in core.